### ISSUE TITLE
run lighthouse against journey site to make adjustments

### ISSUE THREAD

**Issue by btopro** [haxtheweb/issues#2348]
https://journey-chi-kohl.vercel.app/

- [ ] some button complaints about lacking text
- [ ] justified titles
- [ ] button contrast on main page (just make buttons white text)
- [ ] image aspect ratio (tho should make compressions on upload)
